examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,240 @@
+# Steps to to deploy aurora rds cluster
+This example deploys an aurora postgresql database cluster when applied the claim. Below are the steps to execute to
+create a cluster and connect the cluster from a pod with an psql client.
+
+## Pre-requisites
+ - [Upbound AWS Provider Crossplane Blueprint Examples](../../README.md)
+
+
+### create 2 role, one for RDS Monitoring and one for RDS Proxy
+
+Create an IAM role and attach policy to the role (This role is required for aurora to perform monitoring)
+    
+```shell
+    aws iam create-role --role-name aurora-monitoring \
+                            --assume-role-policy-document '{
+            "Version": "2012-10-17",
```

Comment:
Could you please pretty print / format json?

Reply:
Updated as mentioned and moved the json to a different file in the same folder